src/anubhav/assignment8/org/Exceptionthrough.java

```diff
@@ -0,0 +1,22 @@
+package anubhav.assignment8.org;
+public class Exceptionthrough {
```

Comment:
the class name can be like ExceptionThrow

Reply:
Sorry for that, Now I have given some meaningful name to this class and updated the changes suggested by you.